tests/test_search.py

```diff
@@ -965,7 +965,7 @@ def test_aggregations_groupby(client):
 
     res = client.ft().aggregate(req).rows[0]
     assert res[1] == "redis"
-    assert res[3] == ["RediSearch", "RedisAI", "RedisJson"]
+    assert set(res[3]) == {"RediSearch", "RedisAI", "RedisJson"}
```

Comment:
So much smarter than what I was thinking. But, I'm not convinced it always works - in older python versions.

Instead can we do an assert on each item in res[3]?